pkg/api/server/register_containers.go

```diff
@@ -1194,11 +1194,22 @@ func (s *APIServer) registerContainersHandlers(r *mux.Router) error {
 	//       - removing
 	//       - stopping
 	//    description: "Conditions to wait for. If no condition provided the 'exited' condition is assumed."
+	//  - in: query
+	//    name: interval
+	//    type: string
+	//    default: "250ms"
+	//    description: Time Interval to wait before polling for completion.
 	// produces:
 	// - application/json
```

Comment:
Shouldn't this be one or the other? @jwhonce

Reply:
@mheon I do not understand your question.